core-services/prow/02_config/open-cluster-management-io/governance-policy-addon-controller/_prowconfig.yaml

```
@@ -12,8 +12,11 @@ branch-protection:
                 required_approving_review_count: 1
               required_status_checks:
                 contexts:
-                - KinD tests (1.18, v1.19.16)
-                - KinD tests (1.18, latest)
+                - KinD tests / KinD tests (latest, true)
+                - KinD tests / KinD tests (latest, false)
+                - KinD tests / KinD tests (v1.19.16, true)
+                - KinD tests / KinD tests (v1.19.16, false)
+                - Linting and Unit tests / Linting and Unit tests
```

I'm not sure why, but these all get identified inside GitHub with just the trailing name.
```suggestion
                - KinD tests (latest, true)
                - KinD tests (latest, false)
                - KinD tests (v1.19.16, true)
                - KinD tests (v1.19.16, false)
                - Linting and Unit tests
```